### PR TITLE
feat: setup an app for Nest

### PR DESCRIPTION
**Pull request by jupyterjazz**
**Overview**

- Creates a new NOW app for Nest
- Overrides only basic functions - name, description and is_enabled

Would like to get feedback on the naming

### CODE REVIEW

**Review by violenil — APPROVED** [review]
LGTM! Remember to update the changelog :)

File: now/apps/text_to_text_and_image/app.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+    @property
+    def pre_trained_embedding_size(self) -> Dict[Qualities, int]:
+        """
+        Returns a dictionary which maps given quality to embedding size of pretrained model.
+        """
+        return {}
+
+    @property
+    def example_datasource(self) -> List[DemoDatasource]:
```
Comment (joschkabraun) [inline now/apps/text_to_text_and_image/app.py:116]: can remove this function as it is handled for all classes in base class

File: now/apps/text_to_text_and_image/app.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        The interface is as follows:
+        - if kwargs['finetuning']=True, choose finetuning flow
+        - if kwargs['encode']=True, choose encoding flow (to get embeddings for finetuning)
+        - temporarily introduced kwargs['dataset_len'], if app optimizes different flows to it
+        """
+        flow_dir = os.path.abspath(os.path.join(__file__, '..'))
+        self.flow_yaml = os.path.join(flow_dir, 'flow.yml')
+
+    @property
+    def bff(self) -> Optional[str]:
```
Comment (joschkabraun) [inline now/apps/text_to_text_and_image/app.py:64]: would remove this function as it isn't really used yet

File: now/apps/text_to_text_and_image/app.py [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+        """
+        TODO This function is currently not used but already introduces the concept of custom bff
+        Used to configure the bff which is used to define input and output format.
+        :return: the path to the file where the bff routs are configured
+        """
+        return None
+
+    @property
+    def playground(self) -> Optional[str]:
```
Comment (joschkabraun) [inline now/apps/text_to_text_and_image/app.py:73]: would remove this function as it isn't really used yet

File: now/apps/text_to_text_and_image/app.py [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+        Recommended memory limit for the docker client to run this app.
+        """
+        return 8
+
+    @property
+    def finetune_datasets(self) -> [Tuple]:
+        """Defines the list of demo datasets which are fine-tunable."""
+        return ()
+
+    def _check_docker_mem_limit(self) -> bool:
```
Comment (joschkabraun) [inline now/apps/text_to_text_and_image/app.py:144]: can remove as Base app handles this already for all classes

File: now/apps/text_to_text_and_image/app.py [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+                formatter_class=formatter,
+            )
+            for option in self.options:
+                parser.add_argument(
+                    f'--{option["name"]}',
+                    help=option['description'],
+                    type=str,
+                )
+
+    def _check_requirements(self) -> bool:
```
Comment (joschkabraun) [inline now/apps/text_to_text_and_image/app.py:180]: also not needed yet

File: now/apps/text_to_text_and_image/app.py [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+        """
+        Returns true if all requirements on the system are satisfied. Else False.
+        """
+        return True
+
+    def run_checks(self, user_input: UserInput) -> bool:
```
Comment (joschkabraun) [inline now/apps/text_to_text_and_image/app.py:186]: also handled in the base app

**Review by joschkabraun — CHANGES_REQUESTED** [review]
I would remove any attribute & functions which isn't explicitly set in this class as their implementations are all handled in the `Base` app. So would only implement `__init__`, `app_name`, `is_enabled`, `description`

**Review by joschkabraun — APPROVED** [review]
Nice PR LGTM! :rocket:

**Pull request merged**